File: src/test/java/com/example/screenplay/action/LookUpAllUnsuccessfulOutcomes.java
package com.example.screenplay.action;

import net.serenitybdd.screenplay.Actor;
import net.serenitybdd.screenplay.Performable;
import net.serenitybdd.screenplay.actions.Click;

public class LookUpAllUnsuccessfulOutcomes implements Performable {
  @Override
  public <T extends Actor> void performAs(T t) {
    t.attemptsTo(Click.on(".ToggleSideMenu"));
    t.attemptsTo(Click.on(".traceErrors"));

  }
}

File: src/test/kotlin/com/example/screenplay/action/LookUpAllUnsuccessfulOutcomes.kt
package com.example.screenplay.action

import net.serenitybdd.screenplay.Actor
import net.serenitybdd.screenplay.Performable
import net.serenitybdd.screenplay.actions.Click

open class LookUpAllUnsuccessfulOutcomes : Performable {
  override fun <T : Actor> performAs(actor: T) {
    actor.attemptsTo(Click.on(".ToggleSideMenu"))
    actor.attemptsTo(Click.on(".traceErrors"))
  }
}